cmake/Doxyfile.in

```diff
@@ -956,7 +956,7 @@ EXAMPLE_RECURSIVE      = NO
 # that contain images that are to be included in the documentation (see the
 # \image command).
 
-IMAGE_PATH             =
```

Comment:
This file is part of the project template, This could cause annoying merge problems.

Reply:
ok, I now reverted that change and did a separate commit in project-template